src/ralph/deployment/urls.py

```diff
@@ -0,0 +1,26 @@
+from django.conf.urls import url
+
+from ralph.deployment.views import done_ping, files, ipxe, kickstart
+
+urlpatterns = [
+    url(
+        r'^boot.ipxe$',
```

Comment:
could we expose pxe file for each deployment too if we later manage this to work in DHCP server? I mean `'^(?P<deployment_id>[-\w]+)/ipxe$`